packages/slice-machine/src/features/auth/LogoutButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { IconButton, Tooltip } from "@prismicio/editor-ui";
+import { toast } from "react-toastify";
+
+import { logout as managerLogout } from "@/apiClient";
+import { invalidateActiveEnvironmentData } from "@/features/environments/useActiveEnvironment";
+import { invalidateEnvironmentsData } from "@/features/environments/useEnvironments";
+import useSliceMachineActions from "@/modules/useSliceMachineActions";
+
+export function LogoutButton() {
```

Comment:
❓ Did you think about `useServerState`, `Sentry.setUser` for example?

Reply:
I agree with you, I hate this hook
Let's do that yes, I guess if they logout we should forget who they are, even for Sentry call.